spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-ai/src/test/java/com/alibaba/cloud/ai/tongyi/audio/transcription/TongYiAudioTranscriptionClientTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.ai.tongyi.audio.transcription;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import com.alibaba.cloud.ai.tongyi.audio.transcription.api.AudioTranscriptionPrompt;
+import com.alibaba.cloud.ai.tongyi.audio.transcription.api.AudioTranscriptionResponse;
+import com.alibaba.dashscope.audio.asr.transcription.Transcription;
+import com.alibaba.dashscope.audio.asr.transcription.TranscriptionResult;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author: xYLiu
+ * @date: 2024/5/12
+ */
+
+public class TongYiAudioTranscriptionClientTest {
```

Comment:
test class shoule not `public`, is `default`

Reply:
done